pkgs/development/libraries/xapian/bindings/python.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, pkgconfig, xapian, python, pythonPackages
+, zlib, buildPythonPackage }:
+
+buildPythonPackage rec {
+  name = "xapian-bindings-python-${version}";
+  inherit (xapian) version;
+
+  src = fetchurl {
+    url =
+    "https://oligarchy.co.uk/xapian/${version}/xapian-bindings-${version}.tar.xz";
+    sha256 = "0jyym158hhis341lqk5rj9733ki3mcbzbvvzxr2f9fk48zjpf8x6";
+  };
+
+  buildInputs = [ xapian python pythonPackages.sphinx zlib ];
+  nativeBuildInputs = [ pkgconfig ];
+
+  configurePhase = ''
+    ./configure PYTHON=${python}/bin/${python.executable} --prefix=$out PYTHON_LIB="$out/lib/${python.libPrefix}"
+  '';
+
+  buildPhase = ''
+    mkdir -p $out/lib/${python.libPrefix}
+    make
+  '';
+
+  installPhase = ''
+    make install
+  '';
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Python Bindings for Xapian";
+    homepage = https://xapian.org/;
+    license = licenses.gpl2Plus;
+    platforms = platforms.unix;
```

Comment:
we don't have a hard rule for this, but is there a reason you don't want to be a maintainer of this? :)